### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by rusko124**
Continue for #1

swap information:
```
total           used        free      shared  buff/cache   available
Swap:            0B          0B          0B

```
Unfortunately I have a error at the end of the build (Apparently the problem is related to the pi-gen itself.), this error does not always occur that is strange. 
But if you re-run everything okay.
I re-run like this
```
$ CONTINUE=1 PRESERVE_CONTAINER=1 ./build-docker.sh
```

I ran this image and everything worked fine.

### CODE REVIEW

**Comment by vpetersson** [comment]
@rusko124 Note that I reverted the previous PR so you need to re-add those changes (with the comment there).

File: stage2/01-sys-tweaks/00-patches/series [hunk 1]
```diff
@@ -1,4 +1,3 @@
 01-useradd.diff
 04-inputrc.diff
 05-path.diff
-07-resize-init.diff
```
Comment (vpetersson) [inline stage2/01-sys-tweaks/00-patches/series]: Why did you remove this? That's important for auto-resizing the disk on first boot.

Comment (rusko124) [reply stage2/01-sys-tweaks/00-patches/series]: Hmm, you're right, it's unnecessary.

File: export-image/prerun.sh [hunk 2]
```diff
@@ -48,8 +48,9 @@ ROOT_OFFSET=$(echo "$PARTED_OUT" | grep -e '^ 2'| xargs echo -n \
 ROOT_LENGTH=$(echo "$PARTED_OUT" | grep -e '^ 2'| xargs echo -n \
 | cut -d" " -f 4 | tr -d B)
 
-BOOT_DEV=$(losetup --show -f -o "${BOOT_OFFSET}" --sizelimit "${BOOT_LENGTH}" "${IMG_FILE}")
-ROOT_DEV=$(losetup --show -f -o "${ROOT_OFFSET}" --sizelimit "${ROOT_LENGTH}" "${IMG_FILE}")
+BOOT_DEV=$(LOOPDEV_DEBUG=all losetup --show -f -o "${BOOT_OFFSET}" --sizelimit "${BOOT_LENGTH}" "${IMG_FILE}")
```
Comment (vpetersson) [inline export-image/prerun.sh:51]: Accidental commit?

File: stage2/03-screenly/01-run.sh [hunk 3]
```diff
@@ -3,13 +3,25 @@
 # Add checkout to production branch later
 
 on_chroot << EOF
+  curl -s https://bootstrap.pypa.io/get-pip.py | python
+
   git clone https://github.com/Screenly/screenly-ose.git /home/pi/screenly
```
Comment (vpetersson) [inline stage2/03-screenly/01-run.sh:8]: Still working on the wrong branch.

File: export-image/prerun.sh (hunk 2 above)
Comment (rusko124) [reply export-image/prerun.sh:51]: This commit allows you to see the error when assembling the docker.

File: stage2/03-screenly/01-run.sh (hunk 3 above)
Comment (rusko124) [reply stage2/03-screenly/01-run.sh:8]: made changes

**Comment by vpetersson** [comment]
Cool. Merging.
